src/frontend/src/utils/iceberg_engine_opts.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2024 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::ffi::OsString;
+use std::iter;
+
+use clap::Parser;
+
+#[derive(Parser, Clone, Debug)]
+pub struct IcebergEngineOpts {
```

Comment:
Use to capture the environment variables needed for the iceberg engine table, but it would be refactored to retrieve data from meta node

Reply:
Sure, I can create another PR first and then refactor this PR.

Reply:
https://github.com/risingwavelabs/risingwave/pull/19594